### PR TITLE
Remove proto from cc and ansible

### PR DESCRIPTION
**Pull request by ijimiji**

### CODE REVIEW

**Review by slntopp — CHANGES_REQUESTED** [review]
- Billing proto is overwritten completely - pls merge gracefully

File: billing/billing.proto [hunk 1]
```diff
@@ -119,8 +126,7 @@ message Transaction {
      Type(SSD/HDD/NVMe/etc)
       }
   */
-  map<string, google.protobuf.Value> meta = 9;
-  Currency currency = 10;
```
Comment (slntopp) [inline billing/billing.proto]: Why should this be removed?

Comment (ijimiji) [reply billing/billing.proto]: 164f47db0a315468a1e37cc58bef678ff0b27148 is literally proto from currency pr

Comment (slntopp) [reply billing/billing.proto]: yeah, but we had the currencies merged, that's why suspicious, wy the field is removed.
so should it or should it not be removed?

Comment (ijimiji) [reply billing/billing.proto]: I renamed some stuff in https://github.com/slntopp/nocloud/pull/482
So https://github.com/slntopp/nocloud-proto/commit/164f47db0a315468a1e37cc58bef678ff0b27148 should be included

**Pull request merged**